controllers/controlplane/kopscontrolplane_controller.go

```diff
@@ -851,7 +888,7 @@ func RegionBySubnet(kopsControlPlane *controlplanev1alpha1.KopsControlPlane) (st
 }
 
 // createOrUpdateInstanceGroup create or update the instance group in kops state
-func (r *KopsControlPlaneReconciler) createOrUpdateInstanceGroup(ctx context.Context, log logr.Logger, kopsClientset simple.Clientset, kopsCluster *kopsapi.Cluster, kopsInstanceGroup *kopsapi.InstanceGroup) error {
+func (r *KopsControlPlaneReconciliation) createOrUpdateInstanceGroup(ctx context.Context, log logr.Logger, kopsClientset simple.Clientset, kopsCluster *kopsapi.Cluster, kopsInstanceGroup *kopsapi.InstanceGroup) error {
```

Comment:
Since you moved to the new Reconciliation struct, you should also move to use the log inside the struct, as you did on the other cases.